includes/apple-exporter/components/class-slug.php

```diff
@@ -0,0 +1,147 @@
+<?php
+/**
+ * Publish to Apple News: \Apple_Exporter\Components\Slug class
+ *
+ * @package Apple_Exporter
+ */
+
+/**
+ * A slug.
+ *
+ * @since 0.2.0
+ */
+namespace Apple_Exporter\Components;
+
```

Comment:
Update doc comments to match current standards:
```suggestion
/**
 * Publish to Apple News: \Apple_Exporter\Components\Slug class
 *
 * @package Apple_News
 * @subpackage Apple_Exporter\Components
 */

namespace Apple_Exporter\Components;

/**
 * Represents an article slug (a word or phrase that appears near the article title).
 *
 * @since 2.2.0
 */
```